x-pack/packages/kbn-elastic-assistant/impl/connectorland/connector_setup/index.tsx

```diff
@@ -103,11 +103,13 @@ export const useConnectorSetup = ({
 
   // Once streaming of previous message is complete, proceed to next message
   const onHandleMessageStreamingComplete = useCallback(() => {
-    const timeoutId = setTimeout(() => {
+    setTimeout(() => {
       bottomRef.current?.scrollIntoView({ block: 'end' });
+      if (currentMessageIndex >= MESSAGE_INDEX_BEFORE_CONNECTOR) {
```

Comment:
So I think this'll have a bug where the new enterprise messages would put it greater than `MESSAGE_INDEX_BEFORE_CONNECTOR`, so we we ended up switching to use the message length.